src/Syntax/Verify.hs

```diff
@@ -114,16 +114,23 @@ verifyExpr ex@(Let vs e (_, ty)) = do
   verifyExpr e
 verifyExpr (If c t e _) = traverse_ verifyExpr [c, t, e]
 verifyExpr (App f x _) = verifyExpr f *> verifyExpr x
-verifyExpr m@(Fun (PatParam p) x _) = verifyMatch m (getType p) [Arm p Nothing x]
+verifyExpr m@(Fun (PatParam p) x _) = verifyMatch (const $ pure ()) m (getType p) [Arm p Nothing x]
+verifyExpr m@(Fun (EvParam _) (Match e bs _) _) = do
```

Comment:
oof

Reply:
Yeah. I don't think we'll ever inject `ExprWrapper` here or anything, so it _should_ be fine? Maybe...